src/LibChorus/VcsDrivers/Mercurial/HgRepository.cs

```diff
@@ -1475,10 +1471,28 @@ private static void SetExtensions(IniDocument doc, IEnumerable<KeyValuePair<stri
 			}
 		}
 
-		internal static bool CheckExtensions(IniDocument doc, IEnumerable<KeyValuePair<string, string>> extensionDeclarations)
+		/// <summary/>
+		/// <returns>
+		/// Returns true if the extensions in the repo contain only the extensions declared in the given map.
+		///</returns>
+		internal static bool CheckExtensions(IniDocument doc, Dictionary<string, string> extensionDeclarations)
```

Comment:
Yes, the ini file is only for the specific Hg repo. Hg has a global one, but we don't mess with it.